tools/distrib/python/submit.py

```diff
@@ -40,7 +40,7 @@
 
 # Make the push.
 cmd = ['python', 'setup.py', 'sdist']
-subprocess.call(cmd)
+subprocess.call(cmd, cwd=pkgdir)
```

Comment:
This content change is not mentioned in the commit log message - is it intended or is it stray?

If it is intended, please split this commit into two separate commits. Substantive-content-changes and version-bumb-changes should be isolated from one another.

Also if the content change is intended and separated out into its commit, be sure to include in the commit log message something like "this change was accidentally omitted from (citation to earlier pull request or commit)" if that text does indeed reflect the reality.